internal/httphandler/utils.go

```diff
@@ -136,15 +92,15 @@ func extractVerifiedQueryPayload(w http.ResponseWriter, r *http.Request, queryTy
 	case constants.GetDataProof:
 		blockNum, err := getBlockNum(params)
```

Comment:
Same here

Reply:
Same as above, not needed by the catchup service.

Reply:
Done